src/app/client/index.jsx

```diff
@@ -40,6 +41,7 @@ Meteor.startup(
               <Route path="login" component={LoginContainer}></Route>
               <Route path="register" component={registerContainer}></Route>
               <Route path="forgotpassword" component={Page_ForgotPassword} />
+              <Route path="emailSetting" component={EmailSettingContainer} />
```

Comment:
change this path to config
make a container Config
and put EmailSetting component inside it

bcos there will be other configuration as well